src/docs/components/ComponentSettingsInput.jsx

```diff
@@ -1,9 +1,16 @@
+// @flow
 import React from 'react';
-import PropTypes from 'prop-types';
 import Input from 'form-elements/Input';
 import Checkbox from 'form-elements/Checkbox';
 
-const ComponentSettingsInput = ({values, currentValue = '', onChange}) => {
+type PropsType = {
+  // eslint-disable-next-line flowtype/no-primitive-constructor-types
+  values: Boolean | Number | String,
```

Comment:
?? can it be just boolean etc?

Reply:
no, because we'll need to change logic then. Right now this component accept actual constructors, not strings with their names